user_guide_src/source/tutorial/create_news_items/002.php

```diff
@@ -24,8 +24,8 @@ public function create()
 
         // Checks whether the submitted data passed the validation rules.
         if (! $this->validateData($post, [
-            'title' => 'required|min_length[3]|max_length[255]',
-            'body'  => 'required|min_length[10]|max_length[5000]',
+            'title' => 'required|max_length[255]|min_length[3]',
+            'body'  => 'required|max_length[5000]|min_length[10]',
```

Comment:
Out of curiosity. Why is the order changed here? Is this just your preference or something more?

Reply:
Data that is too large should not be processed and should be discarded as soon as possible. This is because it is riskier to process large data.
For example, passing large data to a regular expression engine could result in a DoS attack.

So it is better to check the max_length first.

However, if the data size is checked first, it may not make much difference if the min_length is first.
But, It is obviously not a good idea to check size after another heavy checking, such as a database search.

Reply:
Thanks. That makes sense from the perspective of good practice.